.gitignore

```diff
@@ -152,3 +152,5 @@ composer.phar
 # Build related
 RoboFile.ini
 
+/nbproject/private/
+/nbproject/
```

Comment:
Files belonging to your individual development environment belong to *your own* global `.gitignore` file, not to the project.